### ISSUE TITLE
awbreezedark does not work properly

### ISSUE THREAD

**Issue by Felix00643298** [undecV/python-awthemes#1]
```
  File "D:\workSpace\Others\ZeroMq GUI\dealer_gui.py", line 60, in __init__
    self.setup_ui()
  File "D:\workSpace\Others\ZeroMq GUI\dealer_gui.py", line 87, in setup_ui
    style.theme_use("awbreezedark")
  File "D:\Anaconda\Lib\site-packages\awthemes\__init__.py", line 43, in theme_use
    self.tk.call("package", "require", f"ttk::theme::{themename}")
_tkinter.TclError: can't read "vars(images)": no such element in array
```

Here is my code:
```python
        style = AwthemesStyle(self.root)

        # Get all avaliable themes.
        print("Available themes:", style.theme_names())

        # Set the theme.
        style.theme_use("awbreezedark")
```

The code can work when I try using `awdark`

**Comment by undecV** [comment]
This issue might be caused by the Python-Awthemes package not being able to locate the `...\i\awthemes` directory after installation. As a result, all scalable themes are unable to function properly.

I’ll continue looking for a solution to this problem. 

In the meantime, you can try copying the `awthemes` folder into your project directory and then importing it. This should make it work as expected.

Running `print(awthemes.__file__)` can show you the location of the package.

Thank you for your feedback!

**Comment by undecV** [comment]
The issue has been resolved in version 0.2.1. Please try updating the package by running the following command:

```
pip install --upgrade python-awthemes
```

Thank you again for your response!
